practical/groovy/background_subtract_batch.groovy

```diff
@@ -187,7 +187,10 @@ image_ids.each() { image_id ->
 
     // Save modified image as OME-TIFF using Bio-Formats
```

Comment:
Not sure here - the exercise 2 in the workflow is asking me to upload a Map Annotation . What do I need a bckgr subtraction for then ?

Reply:
background script won't be included I just noticed that it was not created a file in the tmp dir
So I updated it
I can do that in another PR but the time is not on our side

Reply:
No, no need for extra work, I was just confused, leave as it is please